vendor/mpdf/mpdf/src/Mpdf.php

```diff
@@ -939,7 +942,7 @@ class Mpdf implements \Psr\Log\LoggerAwareInterface
 	private $remoteContentFetcher;
 
 	/**
-	 * @var ImageProcessor
+	 * @var \Mpdf\Image\ImageProcessor
```

Comment:
on line 13 the import is removed
```php
use \dokuwiki\plugin\dw2pdf\DokuImageProcessorDecorator as ImageProcessor;
```

As this is only Phpdocs, it is fine for me to not change the ImageProcessor here.